examples/torch/object_detection/datasets/voc0712.py

```diff
@@ -10,23 +10,18 @@
 # limitations under the License.
 import os
 import os.path
-import sys
 from pathlib import Path
 from pathlib import PurePath
 from typing import Callable, Dict, Optional, Tuple
 
 import cv2
+import defusedxml.ElementTree as ET
 from PIL.Image import Image
 from torch.utils import data
 from torchvision import datasets
 
 from examples.torch.object_detection.utils.augmentations import Compose
 
-if sys.version_info[0] == 2:
```

Comment:
Was this change checked somehow?

Reply:
Now no, it check for python2 that not supported since 2020